dashboard/tsconfig.json

```diff
@@ -23,7 +23,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "strict": true
+    "strict": true,
+    "noFallthroughCasesInSwitch": true
```

Comment:
why this rule? It seems a good safeguard with `switch` loops

Reply:
You're right, I just added it using the migration guides aimed at preserving the same behavior. It can be removed (therefore it defaults to false) and we'll have another good safeguard.

Reply:
I'm confused, if before it was defaulting to `true` (because that's safer) and now it's defaulting to false the change was fine the first time :sweat_smile:

Reply:
Ups, yep, I messed up the true/false meaning on my head... :P 

According to the Typescript docs, `true` does enforce this rule, so, as you state, we should keep the `true`. Thanks for pointing this out!

> TypeScript can reports errors for fall-through cases in switch statement where the case clause is non-empty. This check is turned off by default, and can be enabled using --noFallthroughCasesInSwitch.

With --noFallthroughCasesInSwitch, this example will trigger an error:
``` javascript
switch (x % 2) {
  case 0: // Error: Fallthrough case in switch.
    console.log("even");

  case 1:
    console.log("odd");
    break;
}
```

Reply:
cool, thanks for checking